app/scripts/directives/overview/virtualMachineRow.js

```diff
@@ -97,6 +99,20 @@
     };
   }
 
+  angular.module('openshiftConsole').filter('podUptime', function () {
+    return function (pod) {
+      var computeContainerStartTime = _(_.get(pod, 'status.containerStatuses'))
+        .filter({ name: "compute" })
```

Comment:
We should give the filter a more specific name since it assumes a specific container name for KubeVirt.

Reply:
done